### PR TITLE
feat: print a message when no content found

### PR DESCRIPTION
**Pull request by looztra**
-A message is printed when no content was found, either ` No inputs` or ` No outputs.`  or ` No secrets` .

### CODE REVIEW

File: internal/constants.go [hunk 1]
```diff
@@ -45,18 +45,27 @@
 // InputAutoDocEnd is the end of the input
 var InputAutoDocEnd = fmt.Sprintf(AutoDocEnd, "INPUT")
 
+// Content when no inputs
```
[the remark this answers is not in the document]
Comment (jackton1) [reply internal/constants.go:48]: @looztra Can we update the comments

File: internal/constants.go [hunk 2]
```diff
@@ -45,18 +45,27 @@
...
+// Content when no outputs
+var NoOutputsMessage = "No outputs."
+
 // SecretsAutoDocStart is the start of the secrets
 var SecretsAutoDocStart = fmt.Sprintf(AutoDocStart, "SECRETS")
 
 // SecretsAutoDocEnd is the end of the secrets
 var SecretsAutoDocEnd = fmt.Sprintf(AutoDocEnd, "SECRETS")
 
+// Content when no secrets
```
[the remark this answers is not in the document]
Comment (jackton1) [reply internal/constants.go:66]: Same as above

File: internal/constants.go [hunk 3]
```diff
@@ -45,18 +45,27 @@
...
+// Content when no inputs
+var NoInputsMessage = "No inputs."
+
 // OutputAutoDocStart is the start of the output
 var OutputAutoDocStart = fmt.Sprintf(AutoDocStart, "OUTPUT")
 
 // OutputAutoDocEnd is the end of the output
 var OutputAutoDocEnd = fmt.Sprintf(AutoDocEnd, "OUTPUT")
 
+// Content when no outputs
```
[the remark this answers is not in the document]
Comment (jackton1) [reply internal/constants.go:57]: Same as above

File: internal/constants.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (looztra) [reply internal/constants.go]: done

File: internal/constants.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (looztra) [reply internal/constants.go]: done

File: internal/constants.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (looztra) [reply internal/constants.go]: done

**Comment by jackton1** [comment]
@all-contributors please add @looztra for code and test

File: internal/constants.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (jackton1) [reply internal/constants.go]: Looks good @looztra, I'm going to merge ASAP

**Pull request merged**